packages/web-components/src/select/select.ts

```diff
@@ -0,0 +1,206 @@
+import { attr, FASTElement, observable } from '@microsoft/fast-element';
+import { ARIAGlobalStatesAndProperties } from '@microsoft/fast-foundation';
+import { SelectSize } from './select.options.js';
+
+/**
+ * The base class used for constructing a fluent-slider custom element
+ * @public
+ */
+export class Select extends FASTElement {
```

Comment:
I think this will need to include Form Association, I'd take a look at the FormAssociated stuff we have in FAST Foundation. When we update that mixin, it'll likely update things accordingly.

Reply:
I looked at what Select, TextArea, and TextField were doing in FAST Foundation and copied their implementation. Please let me know if I missed anything or did it incorrectly.

Reply:
What can I do on my end in the browser to test if this is working correctly? I'd like to get better at debugging, but I'm not sure what I need to test for to ensure I have this implemented correctly.

Reply:
@procload you'll want to make sure that the control communicates with the form. For instance if you select a value the form data should hold that value.

Reply:
K, thanks. I attempted to follow what was being done on the text-input and text-area from Fast Foundation, but not sure what I'm missing. I need a hand to get this to work. Can we pair to debug when you have a chance?